src/helpers/buildWalletSections.js

```diff
@@ -237,7 +235,6 @@ const coinEditContextMenu = (
             },
             onPressActionSheet: async index => {
               if (index === 1) {
-                store.dispatch(setOpenSmallBalances(true));
```

Comment:
This is one of 2 places that were still using functions from the "old" open state redux.

This is inside the `coinEditContextMenu` for the balances section.
If I understand correctly, the `RecyclerAssetList` is now only used for the Showcase sheet, which does not need this coin edit functionality, and the new RAL2 has this functionality inside of the `CoinDivider`. In order to not remove additional code, I just removed this line, but I expect that the surrounding code is unused.